logger/metrics/metrics.go

```diff
@@ -0,0 +1,195 @@
+		Subsystem: "MediaType",
+		Name:      "Image",
+		Help:      "Stickerization Requests with Image as Media Type",
+	})
+	isvideoQueued := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "Whatsticker",
+		Subsystem: "MediaType",
+		Name:      "Video",
+		Help:      "Stickerization Requests with Video as Media Type",
+	})
+	isnomediaQueued := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "Whatsticker",
+		Subsystem: "MediaType",
+		Name:      "NoMedia",
+		Help:      "Stickerization Requests with Invalid Media Type ",
+	})
+	isvalidQueued := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "Whatsticker",
+		Subsystem: "Validation",
+		Name:      "Valid",
+		Help:      "Valid Stickerization Requests ",
+	})
+	isinvalidQueued := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "Whatsticker",
+		Subsystem: "Validation",
+		Name:      "Invalid",
+		Help:      "Invalid Stickerization Requests ",
+	})
+
+	countryQueued := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "Whatsticker",
+			Subsystem: "SenderCountry",
+			Name:      "Country",
+			Help:      "Stickerization Request Sender's Country",
+		},
+		[]string{
+			"country",
+		},
+	)
+	return StickerizationGauges{
+		GroupMessagesGauge:   isgroupQueued,
+		PrivateMessagesGauge: isprivateQueued,
+		ImageGauge:           isimageQueued,
+		VideoGauge:           isvideoQueued,
+		InvalidMediaGauge:    isnomediaQueued,
+		CountryGauge:         countryQueued,
+		ValidGauge:           isvalidQueued,
+		InvalidGauge:         isinvalidQueued,
+	}
+}
+
+func NewRegistry() *prometheus.Registry {
+	return prometheus.NewRegistry()
+}
+
+func Initialize(registry *prometheus.Registry, gauges StickerizationGauges) MetricConsumer {
+	registry.MustRegister(
+		gauges.CountryGauge,
+		gauges.GroupMessagesGauge,
+		gauges.PrivateMessagesGauge,
+		gauges.ImageGauge,
+		gauges.VideoGauge,
+	)
+	pusher := push.New("http://metrics:9091", "StickerizationRequestMetrics").Gatherer(registry)
+
+	return MetricConsumer{
+		Registry: registry,
+		Gauges:   gauges,
+		Pusher:   pusher,
+	}
+
+}
+
+func PushToGateway(pusher *push.Pusher) {
+	if err := pusher.Add(); err != nil {
+		fmt.Println("Could not push to Pushgateway:", err)
+	}
+	fmt.Println("Metrics Pushed to Pushgateway:", pusher)
+}
+
+func CheckAndIncrementMetrics(stickerMetric StickerizationMetric, stickerGauges *StickerizationGauges) {
+	senderCountry := extractCountry(stickerMetric.MessageSender)
+	if senderCountry != "" {
+		stickerGauges.CountryGauge.With(prometheus.Labels{"country": senderCountry}).Inc()
+	}
+
+	if stickerMetric.IsGroupMessage {
+		stickerGauges.GroupMessagesGauge.Inc()
+	} else {
+		stickerGauges.PrivateMessagesGauge.Inc()
+	}
+
+	if stickerMetric.MediaType == "image" {
+		stickerGauges.ImageGauge.Inc()
+	} else if stickerMetric.MediaType == "video" {
+		stickerGauges.VideoGauge.Inc()
+	} else {
+		stickerGauges.InvalidMediaGauge.Inc()
+	}
+
+	if stickerMetric.Validation {
```

Comment:
Nit: Change to `Validated` instead of `Validation`

Reply:
Okay... will doo